schema/spacewalk/upgrade/susemanager-schema-4.3.3-to-susemanager-schema-4.3.4/003-pillar-upgrade.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO rhnTaskQueue (org_id, task_name, task_data)
```

Comment:
With uyuni released with susemanager-schema version 4.3.4 this should now be 4.3.4-4.3.5.